src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
btw, I can make these changes during the merge if you just want to address the question about whether the geometry should ever be a multi shape or just a always a LineString. Thanks!